src/logging_config/config.json

```
@@ -3,11 +3,11 @@
   "disable_existing_loggers": false,
   "formatters": {
     "simple": {
-      "format": "%(asctime)s │ %(levelname)-7s │ %(name)-17s │ %(message)s",
+      "format": "%(asctime)s %(levelname)-7s %(name)-8s %(message)s",
```

```suggestion
      "format": "%(asctime)s | %(levelname)-7s | %(name)-8s | %(message)s",
```

Much prefer with bars myself, way easier to read? (how small is your screen?)